be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -241,18 +240,17 @@ void OlapScanNode::_scanner_thread(TabletScanner* scanner) {
             chunk = _chunk_pool.pop();
         }
         DCHECK_EQ(chunk->num_rows(), 0);
-        status = scanner->get_chunk(_runtime_state, chunk);
+        status = scanner->get_chunk(_runtime_state, chunk.get());
         if (!status.ok()) {
             QUERY_LOG_IF(ERROR, !status.is_end_of_file()) << status;
             std::lock_guard<std::mutex> l(_mtx);
-            _chunk_pool.push(chunk);
+            _chunk_pool.push(std::move(chunk));
```

Comment:
why use move?

Reply:
After pushing `chunk` to `_chunk_pool`, we will break this loop, which means `chunk` will never be used by others. Therefore, we can use `move` here. And using `move` here can use move constructor instead of copy constructor of `shared_ptr`.